### PR TITLE
Charts for maintenance windows in Provisioner

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Charts changes and image bump

**Related issue(s)**

### CODE REVIEW

File: resources/compass/charts/provisioner/values.yaml [hunk 1]
```diff
@@ -11,14 +10,16 @@ deployment:
   strategy: {} # Read more: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
 
 security:
-  skipTLSCertificateVeryfication: false
+  skipTLSCertificateVeryfication: true
```
Comment (akgalwas) [inline resources/compass/charts/provisioner/values.yaml:13]: Is this change vital?

Comment (Maladie) [reply resources/compass/charts/provisioner/values.yaml]: Just a mistake

File: resources/compass/charts/provisioner/templates/deployment.yaml [hunk 2]
```diff
@@ -116,10 +118,14 @@ spec:
...
               readOnly: true
+        {{- end }}
+        {{if .Values.gardener.maintenanceWindowConfigMapName }}
+            - mountPath: /gardener/maintenance
+              name: gardener-maintenance-config
+              readOnly: true
         {{- end }}
             - mountPath: /gardener/kubeconfig
               name: gardener-kubeconfig
-              readOnly: true
```
Comment (Szymongib) [inline resources/compass/charts/provisioner/templates/deployment.yaml:122]: Should it be removed?

File: resources/compass/charts/provisioner/values.yaml [hunk 3]
```diff
@@ -1,5 +1,4 @@
 deployment:
-  replicaCount: 1
```
Comment (Szymongib) [inline resources/compass/charts/provisioner/values.yaml:2]: Same here, should it be removed?

**Pull request merged**